### DISCUSSION TITLE
why 3.5.5-p1 realease donot support maven-groovy

### DISCUSSION THREAD

**Discussion by zanegg** [naver/ngrinder#833] [Q&A]
why 3.5.5-p1 realease donot support groovy maven project

**Comment by donggyu04** [comment]
@zanegg 

groovy gradle script is supported instead of groovy maven.
but, only blocks the creation of groovy maven  script, execution is still possible.
(make sure install maven manually and added it to your PATH)

there was no reason to support groovy maven while supporting groovy gradle.
